### PR TITLE
Add a recipe for AeHMC

### PR DESCRIPTION
**Pull request by brandonwillard**
This PR adds a recipe for [AeHMC](https://github.com/aesara-devs/aehmc).

~The build probably won't succeed until https://github.com/conda-forge/staged-recipes/pull/15766 goes through.~

### CODE REVIEW

**Comment by brandonwillard** [comment]
@conda-forge/help-python

File: recipes/aehmc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python >=3.6
+    - numpy >=1.18.1
+    - scipy >=1.4.0
+    - aesara >=2.1.1
+    - aeppl >=0.0.5
+
+test:
+  imports:
+    - aehmc
```
Comment (dopplershift) [inline recipes/aehmc/meta.yaml:31]: It looks like `import aehmc` doesn't actually do anything meaningful. Can you add a few more import tests to validate that the package isn't broken?

**Review by dopplershift — COMMENTED** [review]
Recipe is in good shape. Need confirmation from @twiecki and @rlouf that they are willing to be listed as recipe maintainers.

**Comment by twiecki** [comment]
Confirmed.

File: recipes/aehmc/meta.yaml (hunk 1 above)
Comment (brandonwillard) [reply recipes/aehmc/meta.yaml:31]: Good point; @rlouf, which modules should we import?

**Comment by rlouf** [comment]
Confirmed.

File: recipes/aehmc/meta.yaml (hunk 1 above)
Comment (rlouf) [reply recipes/aehmc/meta.yaml:31]: `aehmc.hmc` and `aehmc.nuts` would makje sense.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**